sdk/keyvault/Azure.Security.Keyvault.Keys/src/DeletedKey.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Security.Keyvault.Keys
```

Comment:
Is this the right `Keyvault` casing? Same for the project name.

Reply:
Ohh thanks.